src/dev/src/app/datagrid/selection-single/selection-single.html

```diff
@@ -126,7 +126,7 @@ <h3>Server-driven, trackBy item</h3>
   <p class="card-text username-list">
     Selected user:
     <em *ngIf="!trackByIdSingleSelected">No user selected.</em>
-    <span class="username" *ngIf="trackByIdSingleSelected">{{trackByIdSingleSelected.name}}</span>
+    <span class="username" *ngIf="trackByIdSingleSelected">{{trackByIdServerSingleSelected.name}}</span>
```

Comment:
You need to update the other properties here on lines 128-129 to `trackByIdServerSingleSelected`

Reply:
Thank you!